tests/integration/test_subscriptions_cli.py

```diff
@@ -13,14 +13,33 @@
 
 """
 
+import asyncio
 import json
+import uuid
 
 from click.testing import CliRunner
 import pytest
 
 from planet.cli import cli
 import planet.cli.subscriptions
-from planet.cli.subscriptions import _count_fake_subs
+from planet.cli.subscriptions import PlaceholderSubscriptionsClient
+
+
+@pytest.fixture
+def subscription_count():
+    """Return a function that counts subscriptions in the system."""
+
+    def func():
+
+        async def count_em():
+            return len([
+                sub async for sub in
+                PlaceholderSubscriptionsClient().list_subscriptions()
+            ])
+
+        return asyncio.run(count_em())
+
+    return func
```

Comment:
Being able to check the count of fake subscriptions in the system provides a sanity check in a few tests. I wrote a fixture that uses the placeholder client within an event loop to do so. This won't work for the real Subscriptions API, so we'll eventually delete this fixture and all usage.